### DISCUSSION TITLE
Images don't import in Vercel build.

### DISCUSSION THREAD

**Discussion by trevorwrightdev** [vercel/next.js#36173] [Help]
When I make a build of my website, all of the images that utilize the img tag do not get imported. Is there a way I can use an img tag in a build? I need to use the image-rendering: pixelated attribute because my site uses pixel art. When using the img tags the resolution is nice and crisp, but switching to the Next Image tag makes the image look less vibrant, even with quality at 100. Is there a way I can import good pixel art using the Next Image tag, or better yet, a way I can use img tags with no problem?

**Comment by icyJoseph** [comment]
Could you pass [unoptimized](https://nextjs.org/docs/api-reference/next/image#unoptimized), set to true, to your `next/image` and see if you still get it as you wish to have it?

> When true, the source image will be served as-is instead of changing quality, size, or format. Defaults to false.
